client/src/components/showMeMore/Container.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import Button from '@material-ui/core/Button';
+import ButtonGroup from '@material-ui/core/ButtonGroup';
+import { makeStyles } from '@material-ui/core/styles';
+import { Grid } from '@material-ui/core';
+import Observations from './observations';
+import AvgTemperature from '../AvgTemperature';
+import Chart from '../Chart';
+import chartType from '../../utils/chartType';
+import BehavioralChart from '../behavior/chart/BehavioralChart';
+import buttonsCss from '../../css/buttons';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    paddingBottom: '15vh',
+  },
+  buttons: {
+    '& > *': {
+      ...buttonsCss.buttons,
+      margin: theme.spacing(1),
+      maxWidth: '100px',
+    },
+  },
+  behaveDiv: {
+    paddingBottom: '30vh',
+    width: '100vw',
+    height: '60vh',
+    [theme.breakpoints.up('md')]: { width: '60vw', height: '70vh' },
+  },
+  feverDiv: {
+    marginBottom: '5%',
+    width: '100vw',
+    height: '50vh',
+    [theme.breakpoints.up('md')]: { width: '60vw' },
+  },
+  export: { ...buttonsCss.buttons, margin: theme.spacing(1) },
+}));
+
+const ShowMeMore = () => {
+  const classes = useStyles();
+  const [value, setValue] = useState('observations');
```

Comment:
We should use something more descriptive here like `[chartType, setChartType]` or something